Add missing fleet images

florinnichifiriuc opened the pull request:
Add missing fleet images.

Commit:
Add missing fleet images

florinnichifiriuc committed:
Update fleet.images.user.js

fl0v commented on fleet.images.user.js:
```diff
@@ -0,0 +1,31 @@
+    const imgExt = '.gif';
+    const replaceImage = function (img) {
+        const [,size,id] = img.src.match(imgIdPattern);
+        if (id.indexOf('holo')>-1 || id =='projector')
```
fix code styling

fl0v commented on fleet.images.user.js:
```diff
@@ -0,0 +1,31 @@
+            {
+            img.style.opacity ='0.5';
+            }
+        let image =img.src.replace('main/ships/holo_','small/').replace('main/ships/projector','small/trader').replace('main/ships/trader','small/trader').replace('projector','trader').replace('holo_','');
```
fara replace inlatuit,
trebuie definita o lista de forma { pattern => new_str, ...}
si executat replace cu RegExp
```
const patternMap = {
    'main\/ships\/holo_' => 'small/'
};
const pattern = new RegExp(Object.keys(patternMap).join("|"),"gi");
img.src = img.src.replace(pattern, (m) => patternMap[m]);
```

fl0v committed:
@downloadUrl trebuie sa fie spre master

fl0v committed:
nodeList suporta forEach

Nu e nevoie de Array.from decat daca se doreste acces la alte  metode de prelucare array
Ex: reduce, sort, map, ...
